composer.json

```diff
@@ -1,16 +1,45 @@
 {
-    "name": "sturgeon/phparse",
+    "name": "phpversions/phparse",
     "version": "0.2.1",
```

Comment:
Should we remove the `version`?

Reply:
Hmmm, I think this could be useful for other people, so maybe not remove but definitely bump?

Reply:
Because we're using Git tags, which Composer/Packagist will read version numbers from, it should be safe to omit this (see https://getcomposer.org/doc/04-schema.md#version).

Maybe we can set a [branch alias](https://getcomposer.org/doc/articles/aliases.md#branch-alias) instead?  We do that with League packages: https://github.com/thephpleague/commonmark/blob/master/composer.json#L68

Reply:
that sounds like a good idea.